db/migrate/20221114050818_create_bookings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateBookings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :bookings do |t|
+      t.string :status
```

Comment:
this should be an integer, we are going to use the "Enums" thing that trouni send to us